queries/loans/types.ts

```diff
@@ -0,0 +1,16 @@
+import { Big } from 'utils/formatters/big-number';
```

Comment:
Could we use our BigNumber library instead?

Reply:
Switched. Wish bignumber.js had the same API as ethers.BigNumber, similar to big.js though.